qualtran/_infra/bloq.py

```diff
@@ -515,3 +515,24 @@ def wire_symbol(self, soq: 'Soquet') -> 'WireSymbol':
         from qualtran.drawing import directional_text_box
 
         return directional_text_box(text=soq.pretty(), side=soq.reg.side)
+
+    def get_builder_args(self) -> Dict[str, Any]:
```

Comment:
I've changed the method to "get_kwargs(self)->Dict[str, Any]. This method must be a public method since it is called by _bloq_args_to_proto (which is outside the bloq instance).  Please let me know if this resolves the issue or if we should use a better name.